multi-node/aws/pkg/config/config_test.go

```diff
@@ -77,6 +82,8 @@ controllerIP: 10.4.3.5
 podCIDR: 172.4.0.0/16
 serviceCIDR: 172.5.0.0/16
 dnsServiceIP: 172.6.100.101 #dnsServiceIP not in service CIDR
+`, `
+routeTableId: rtb-xxxxxx # routeTableId specified without vpcId
```

Comment:
All these strings are appended to [`MinimalYamlConfig`](https://github.com/colhom/coreos-kubernetes/blob/existing-vpc/multi-node/aws/pkg/config/config_test.go#L8) before parsing, which has all the mandatory config options set.

This validity of `MinimalYamlConfig` itself is tested [here](https://github.com/colhom/coreos-kubernetes/blob/existing-vpc/multi-node/aws/pkg/config/config_test.go#L17)